src/client/src/modals/ViewPlatformRequirementsModal/messages.ts

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  developmentRequirements: {
+    id: "ViewPlatformRequirementsModal.developmentRequirements",
+    defaultMessage: "Development Requirements",
+  },
+  ariaCloseModalLabel: {
+    id: "ViewPlatformRequirementsModal.ariaCloseModalLabel",
+    defaultMessage: "Close Platform Requirements Modal",
+  },
+  checkAndInstallRequirements: {
```

Comment:
Not sure about those 2 resources... maybe it´s better having just one 
`"To check and install missing depevelopment requirements {0}."`
and the other one 
`"review the docs"`

And then doing the replace **{0}** by the `<a... {review the docs } ... >`

Reply:
Split in text and bottom link.